akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -35,6 +35,49 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem)
 
   val http = Http(system)
 
+  /**
+   * Handle requests using HTTP/2 immediately, without any TLS or negotiation layer.
+   */
+  def bindAndHandleRaw(
+    handler:   HttpRequest ⇒ Future[HttpResponse],
+    interface: String, port: Int = DefaultPortForProtocol,
+    settings:    ServerSettings = ServerSettings(system),
+    parallelism: Int            = 1,
+    log:         LoggingAdapter = system.log)(implicit fm: Materializer): Future[ServerBinding] = {
+    val effectivePort = if (port >= 0) port else 80
+
+    val serverLayer: Flow[ByteString, ByteString, Future[Done]] = Flow.fromGraph(StreamUtils.fuseAggressive(
```

Comment:
we can remove `StreamUtils.fuseAggressive` now since Akka HTTP 10.1 requires 2.5 and this is a No-Op on 2.5+